robot/bullet.xacro

```diff
@@ -12,13 +12,13 @@
 
 	<gazebo>
 		<plugin name="gazebo_ros_control" filename="libgazebo_ros_control.so">
-			<robotNamespace>/syropod</robotNamespace>
+			<robotNamespace></robotNamespace>
```

Comment:
I suspect there's a chance this may explicitly clear the namespace. This could adversely affect specifying a namespace upstream.

Reply:
Removed